packages/main/test/sap/ui/webcomponents/main/pages/ShellBar.html

```diff
@@ -29,58 +29,155 @@
+		profile="https://openui5nightly.hana.ondemand.com/test-resources/sap/f/images/Woman_avatar_01.png"
 		second-title="Second Title"
-		notification-count="42"
-		show-profile
-		show-notification
+		notification-count="99+"
+		show-notifications
 		show-product-switch
-		show-search
 		show-co-pilot
 	>
-		<ui5-shellbar-item id="disc" src="sap-icon://disconnected" text="Disconnect"></ui5-shellbar-item>
-		<ui5-shellbar-item id="call" src="sap-icon://incoming-call" text="Incoming Calls"></ui5-shellbar-item>
 
-		<ui5-input data-ui5-slot="input"></ui5-input>
+		<ui5-icon src="nav-back" data-ui5-slot="icon"></ui5-icon>
+
+		<ui5-shellbar-item data-ui5-slot="items" id="disc" src="sap-icon://disconnected" text="Disconnect"></ui5-shellbar-item>
+		<ui5-shellbar-item data-ui5-slot="items" id="call" src="sap-icon://incoming-call" text="Incoming Calls"></ui5-shellbar-item>
+
+		<ui5-input data-ui5-slot="searchField"></ui5-input>
 	</ui5-shellbar>
 
-	<script>
-		document.getElementById("disc").addEventListener("press", event => {
-			console.log(event.target);
-		});
+	<ui5-popover id="popover" hide-header placement-type="Bottom">
+		<div class="popover-header">
+			<ui5-title style="padding: 0.25rem 1rem 0rem 1rem">John Doe</ui5-title>
+		</div>
+
+		<div class="popover-content">
+			<ui5-list separators="None" >
+				<ui5-li type="Active"icon="sap-icon://sys-find">App Finder</ui5-li>
+				<ui5-li type="Active"icon="sap-icon://settings">Settings</ui5-li>
+				<ui5-li type="Active"icon="sap-icon://edit">Edit Home Page</ui5-li>
+				<ui5-li type="Active"icon="sap-icon://sys-help">Help</ui5-li>
+				<ui5-li type="Active"icon="sap-icon://log">Sign out</ui5-li>
+			</ui5-list>
+		</div>
+	</ui5-popover>
 
-		document.getElementById("call").addEventListener("press", event => {
-			console.log(event.target);
+	<ui5-popover id="app-list-popover" hide-header placement-type="Bottom">
+		<ui5-list separators="None">
+			<ui5-li type="Active">Application 1</ui5-li>
+			<ui5-li type="Active">Application 2</ui5-li>
+			<ui5-li type="Active">Application 3</ui5-li>
+			<ui5-li type="Active">Application 4</ui5-li>
+			<ui5-li type="Active">Application 5</ui5-li>
+		</ui5-list>
+	</ui5-popover>
+
+	<ui5-input id="press-input" style="margin-top: 2rem; width: 240px;">
+
+	</ui5-input>
+
+	<script>
+		shellbar.addEventListener("profilePress", function(event) {
+			popover.openBy(event.detail.iconRef);
+			window["press-input"].value = "Profile";
 		});
 
-		document.getElementById("shellbar").addEventListener("notificationPress", event => {
-			console.log("notifications");
+		shellbar.addEventListener("titlePress", function(event) {
+			window["app-list-popover"].openBy(event.detail.iconRef);
+			window["press-input"].value = "Title";
 		});
 
-		document.getElementById("shellbar").addEventListener("profilePress", event => {
-			console.log("profile");
+		shellbar.addEventListener("notificationsPress", function(event) {
+			window["press-input"].value = "Notifications"
```

Comment:
check if the sample works on ie and edge  just in case as I have feeling that the id of an element is not assigned to the window object in one of these browsers, just check if everything is fine.